frontend/src/views/ProducerFormPage/ProductStep.vue

```diff
@@ -54,15 +54,43 @@
     <v-icon class="mr-1" name="ri-capsule-fill" />
     Format
   </h2>
-  <DsfrInputGroup class="mt-6 max-w-md">
-    <DsfrSelect
-      label="Forme galénique"
-      v-model="payload.galenicFormulation"
-      :options="galenicFormulation"
-      :required="true"
-    />
-  </DsfrInputGroup>
   <div class="grid grid-cols-2 gap-4">
+    <DsfrFieldset legend="Forme galénique" legendClass="fr-label !font-normal !pb-0">
+      <div class="flex">
+        <div class="max-w-32">
+          <DsfrSelect
+            :options="formulationStates"
+            v-model="payload.galenicFormulationState"
```

Comment:
Est-ce qu'on veut que l'état soit sauvegardé en base dans le modèle déclaration ? (est-ce le cas aujourd'hui sur téléicare ?) À doigt mouillé je dirais non - vu qu'on a l'info sur l'état de la formulation galénique dans son propre modèle avec `is_liquid`, donc pas besoin de le mettre dans le payload. On peut avoir une ref `galenicFormulationState` côté frontend seulement non ?

Reply:
aujourd'hui ce n'est pas stocké dans téléIcare (c'est une nouveauté inédite de Compl'Alim :sun_with_face: ).
Tu as raison concernant la payload : pas besoin de stocker l'état dedans cas pas besoin de stocker en base.
Je fais la modif : afa067d4e5e0e16edbf969fc942a03dd6938369a